data/json/recipes/armor/head.json

```diff
@@ -1095,7 +1088,7 @@
     "components": [
       [ [ "hat_hard", 1 ], [ "helmet_bike", 1 ] ],
       [ [ "flashlight", 1 ] ],
-      [ [ "string_36", 1 ], [ "string_6", 6 ], [ "duct_tape", 10 ], [ "medical_tape", 20 ] ]
+      [ [ "string_36", 1 ], [ "string_6", 6 ], [ "withered", 12 ], [ "straw_pile", 12 ], [ "duct_tape", 10 ], [ "medical_tape", 20 ] ]
```

Comment:
Hard to find how this is used because the diff size is too large, it will not display the actual changes to the food file in that pull request.